tests/config/test_data.config

```diff
@@ -3,7 +3,7 @@
 
 params {
     // Base directory for test data
-    test_data_base = "https://raw.githubusercontent.com/nf-core/test-datasets/modules"
+    test_data_base = "https://raw.githubusercontent.com/tucano/test-datasets/modules"
```

Comment:
this can't be changed....has to point to official nf-core test datasets repo

Reply:
fixed ... sorry my bad.